src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,224 @@
+* [Your repository's commit sha](#sha)
+* [Your repository's URL](#repo-url)
+* If you're using an APM agent, [configure your APM agent for CodeStream](#apm)
+* [Data being reported to New Relic from an app, host, or other entity ](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==)
+* [A workload for errors inbox](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#create)
+* [An error](/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/#error-details)
+
+New Relic errors inbox is a single place to proactively detect, prioritize, and take action on all the errors before they impact customers. Now via CodeStream, you can jump from an error directly to the code causing the error in your IDE.
+
+<Callout variant="tip">
+    Limited to APM, browser, and mobile errors.
+</Callout>
+
+From **[one.newrelic.com/](https://one.newrelic.com/)**, go to **Errors Inbox**, click a stack trace error, then click **Open in IDE**.
+
+### Connect a stack trace error to the correct version of code [#sha]
+
+For the **Open in IDE** button to open the version of your code that generated the stack trace error, you'll need your [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection)
+and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.  
+
+CodeStream only needs the first seven characters of your commit sha (for example, `734713b`) to make this connection, but you can include the entire sha.  
+
+Alternately, you can use a release tag (such as `v0.1.209` or `release-209`) for CodeStream to find the correct version of your code.
+
+### Configure your New Relic APM [#apm]
+
+For New Relic APM, the commit and/or release tag (`tags.commit` and `tags.releaseTag`) are added as attributes on `Transaction` and `TransactionError` events. The APM agent protocol provides an easy way to set this up via environment variables.  
+
+We recommend setting one or both of these variables as part of your build pipeline.
+* `NEW_RELIC_METADATA_COMMIT` - The commit sha. You can include the whole thing or only the first seven characters.
+* `NEW_RELIC_METADATA_RELEASE_TAG` - A release tag (such as `v0.1.209` or `release-209`). This has the advantage of being human readable.
+
+### Give us your git repository's URL [#repo-url]
+
+In order to link your data in New Relic to your code, we need to know the URL for your git repository.  
+
+The URL can be in either the SSH or HTTPS format:
+* `git@github.com:newrelic/beta-docs-site.git`
+* `https://github.com/newrelic/beta-docs-site.git`
+
+You can use any one of these methods to set up your repository:
+* Set up via environment variable using New Relic APM. We recommend this method.
+* Use the NerdGraph API.
```

Comment:
Now users can also set them up
<img width="510" alt="Screen Shot 2021-10-13 at 9 47 19 PM" src="https://user-images.githubusercontent.com/47154052/137253748-e126ac78-febd-4b80-a857-3a43c52f348f.png">
 in the UI like this app has: https://staging.onenr.io/0M8jqKLJlwl
<img width="449" alt="Screen Shot 2021-10-13 at 9 47 09 PM" src="https://user-images.githubusercontent.com/47154052/137253729-69e15a89-40ee-4e10-a9cc-bf3f9b0b3762.png">




(and for Browser they can use the source maps api i mentioned in my other comment)